LoopKitUI/Views/GuideButtonStyle.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  GuideButtonStyle.swift
+//  LoopKitUI
+//
+//  Created by Pete Schwamb on 2020-03-04.
```

Comment:
"Created by Pete Schwamb"?   Not a problem, just wondering if you really meant to put "Nathaniel Hamming"?  Or did Pete make this?

Reply:
I stole it from Pete.